aldryn_events/tests/frontend/integration/pages/alteli.js

```diff
@@ -0,0 +1,166 @@
+// ALDRYN TESTS LIBRARY
+// ############################################################################
+'use strict';
+/* global expect, browser, jasmine */
+/* jshint browser: true */
+/* jshint shadow: true */
+
+var aldrynTestsLibrary = {
+    // utility to set browser window size
+    setWindowSize: function (width, height) {
+        browser.driver.manage().window().setSize(
+            width,
+            height
+        );
+    },
+
+    // utility to pre-create folder with 0777 permissions
+    preCreateFolder: function (folderName) {
+        var fs = require('fs');
```

Comment:
no need to require fs in multiple places. should move it to the top of the file instead and reuse same variable